src/main/webapp/WEB-INF/views/template/layout/admin/template.jsp

```diff
@@ -38,7 +28,7 @@
 								<div class="contentsFixBack">
 									<div class="tabMenu">
 										<ul class="nav-tab-menu">
-											<li style="display:none;">
+											<li style=display:none;">
```

Comment:
Please add the missing quotes.

```suggestion
											<li style="display:none;">
```

Reply:
Thanks, I fixed it 👍🏻

Reply:
Please combining two commit([6d7bfcc](https://github.com/fosslight/fosslight/pull/924/commits/6d7bfcc04c33af052e87fdde0a3944255cb43eaf), [258b755](https://github.com/fosslight/fosslight/pull/924/commits/258b7556359328ae14c949d8b346195637f53b36)) into one with squash

Reply:
yep ✓